src/Features/LanguageServer/Protocol/Handler/SemanticTokens/SemanticTokensHelpers.cs

```diff
@@ -169,13 +172,53 @@ public static ClassifiedSpan[] ConvertMultiLineToSingleLineSpans(SourceText text
             }
         }
 
+        internal static async Task<int[]> HandleRequestHelperAsync(
```

Comment:
>HandleRequestHelperAsync [](http://example.com/codeflow?start=42&length=24)

Might be nice if this were moved closer to the top as it's the main entry point to this class now